experimental/dds/tree2/src/shared-tree-core/sharedTreeCore.ts

```diff
@@ -471,12 +358,14 @@ export class SharedTreeCore<TEditor extends ChangeFamilyEditor, TChange> extends
 			0x674 /* Unexpected transaction is open while applying stashed ops */,
 		);
 		const { revision, change } = parseCommit(content, this.changeCodec);
-		const [commit, delta] = this.addLocalChange(change, revision);
-		this.editManager.localBranchUndoRedoManager.trackCommit(
-			commit,
+		this.changeEvents.emit("newLocalChange", change);
+		this.applyingStashedCommit = true;
```

Comment:
This is also a bit silly, but there is no event/state that you can query in the runtime to know "when are stashed ops finished". So, we use this boolean to prevent ourselves from submitting these ones as new ops.